src/main/scala/org/biodatageeks/sequila/apps/SequilaApp.scala

```diff
@@ -0,0 +1,20 @@
+package org.biodatageeks.sequila.apps
+
+import org.apache.spark.sql.{SequilaSession, SparkSession}
+import org.biodatageeks.sequila.utils.SequilaRegister
+
+trait SequilaApp {
+  def createSequilaSession(): SequilaSession = {
+    System.setProperty("spark.kryo.registrator", "org.biodatageeks.sequila.pileup.serializers.CustomKryoRegistrator")
```

Comment:
should it be set explicitly?

Reply:
I think so. If not I suggest putting it into issue and testing it